ci-operator/config/kubevirt/must-gather/kubevirt-must-gather-main.yaml

```diff
@@ -68,7 +68,7 @@ tests:
     - as: test-targeted
       cli: latest
       commands: |
-        oc adm must-gather --image="${KMG_IMAGE}" --dest-dir=${ARTIFACT_DIR}/must-gather-output -- gather_vms_details
+        oc adm must-gather --image="${KMG_IMAGE}" --dest-dir=${ARTIFACT_DIR}/must-gather-output --vms_details
```

Comment:
I think this should be `-- gather --vms_details`, as is `--vms_details` is a flag for `oc adm must-gather` and not for the image  script

Reply:
thanks, amended